robottelo/host_helpers/satellite_mixins.py

```diff
@@ -443,3 +443,25 @@ def api_factory(self):
     @lru_cache
     def ui_factory(self, session):
         return UIFactory(self, session=session)
+
+
+class IoPSetup:
+    """Helper for configuring on prem Insights Advisor engine."""
+
+    def configure_insights_on_prem(self):
+        """Configure on prem Advisor engine on Satellite"""
+        logger.info("Configuring Satellite with local Insights advisor(IoP)")
```

Comment:
Same here and lets use new branding :)
```suggestion
        logger.info('Configuring Satellite with local Red Hat Lightspeed')
```